hivemind/averaging/training.py

```diff
@@ -86,15 +95,17 @@ def step(self, data_lock: Optional[Lock] = None, wait: bool = True, **kwargs):
                     if use_old_local_tensors:
                         # since tensors might have changed, we subtract old_local_tensor and add averaged. This prevents
                         # losing local updates that might have occurred during averaging
-                        for averaged_tensor, local_tensor, old_local_tensor in zip(averaged_tensors, local_tensors,
-                                                                                   old_local_tensors):
-                            local_tensor[...] += averaged_tensor.to(dtype=local_tensor.dtype,
-                                                                    device=local_tensor.device) - \
-                                                 old_local_tensor.to(dtype=local_tensor.dtype,
-                                                                     device=local_tensor.device)
+                        for averaged_tensor, local_tensor, old_local_tensor in zip(
+                            averaged_tensors, local_tensors, old_local_tensors
+                        ):
+                            local_tensor[...] += averaged_tensor.to(
+                                dtype=local_tensor.dtype, device=local_tensor.device
+                            ) - old_local_tensor.to(dtype=local_tensor.dtype, device=local_tensor.device)
                     else:
                         for averaged_tensor, local_tensor in zip(averaged_tensors, local_tensors):
-                            local_tensor[...] = averaged_tensor.to(dtype=local_tensor.dtype, device=local_tensor.device)
+                            local_tensor[...] = averaged_tensor.to(
+                                dtype=local_tensor.dtype, device=local_tensor.device
+                            )
```

Comment:
```suggestion
                            local_tensor[...] = averaged_tensor.to(local_tensor.dtype).to(local_tensor.device)
```
Maybe now you think that I'm a moron. Sorry.